README.md

```
@@ -15,6 +15,11 @@ Follow the steps below to build and view the site locally.
 
 ## Building
 
+* Initialize, update, and clone any submodules within a Git repository
+  ```shell
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)